knowledge-base/grid-checkbox-editing.md

```diff
@@ -26,17 +26,14 @@ res_type: kb
 
 ## Description
 
-To edit a checkbox in a Grid cell, users need to click once to put the row (or cell) in edit mode. They they need another click to change the checkbox value. How to do this with only one click?
+This KB article answers the following questions:
 
-How to update boolean values in the Grid with no edit mode?
-
-How to toggle checkboxes in the Grid directly without using the built-in editing feature?
-
-The Grid is not editable, but it should allow users to check and uncheck check box values.
-
-How to use a single click on Grid checkboxes to set the opposite value?
-
-How to get a checkbox value in the Grid for update action on button click?
+* To edit a checkbox in a Grid cell, users need to click once to put the row (or cell) in edit mode. They they need another click to change the checkbox value. How to do this with only one click?
+* How to update boolean values in the Grid with no edit mode?
+* How to toggle checkboxes in the Grid directly without using the built-in editing feature?
+* The Grid is not editable, but it should allow users to check and uncheck check box values.
+* How to use a single click on Grid checkboxes to set the opposite value?
+* How to get a checkbox value in the Grid for update action on button click?
```

Comment:
Just a suggestion based on our conversation from Teams. If it doesn't make sense, discard it.

```suggestion
* How to get the value of a checkbox and use it for an update action on button click?
```